bakery/src/scripts/fetch_book_group.sh

```diff
@@ -12,5 +12,4 @@ if [[ ! -f "${CONTENT_OUTPUT}/raw/collections/$(cat "${BOOK_INPUT}/slug").collec
 fi
 rm -rf "${CONTENT_OUTPUT}/raw/.git"
 wget "${BOOK_SLUGS_URL}" -O "${CONTENT_OUTPUT}/book-slugs.json"
-mv "${CONTENT_OUTPUT}/raw/media" "${RESOURCE_OUTPUT}/."
```

Comment:
Super minor: Should we `rmdir ${CONTENT_OUTPUT}/raw/media` after invoking `fetch-map-resources` just to clean that up?